src/goto-cc/cl_message_handler.h

```diff
@@ -29,6 +29,16 @@ class cl_message_handlert : public console_message_handlert
     const source_locationt &location) override;
 
   using console_message_handlert::print;
+
+  /// With \p yes set to \c true, prefix warnings with an error message.
+  /// \param yes: Whether or not to prefix warnings.
+  void print_warnings_as_errors(bool yes)
```

Comment:
Do we really need that?

Reply:
I'd say it depends on how closely we'd like to mimic the behaviour of Visual Studio and GCC. Prior to this change, we just printed "warning" but nonetheless failed with a non-zero exit code when `Werror` was used. With this change, we would actually print "error" for such now-fatal warnings, as those compilers do.

Reply:
Feels a bit cosmetic, but why not.